### DISCUSSION TITLE
AnimeIngá 2023: Reserve a data! - AnimeIngá 2023

### DISCUSSION THREAD

**Discussion by giscus** [AnimeInga/animeinga.github.io#11] [General]
# AnimeIngá 2023: Reserve a data! - AnimeIngá 2023

O AnimeIngá é um evento de cultura pop e anime que acontece em Maringá, PR. Reúne fãs de anime, mangá, jogos eletrônicos e cosplay, com atrações como convidados especiais, palestras e concursos.

https://animeinga.com.br/blog/2023/05/18/animeing%C3%A1-2023-reserve-a-data/

**Comment by 0016Simone0016** [comment]
Olá gostaria de saber qual tipo de alimento posso doar? Comprei 2 ingressos 2° lote social
